grid/social-media/followers/profile.supr

```diff
@@ -93,3 +142,11 @@ usecase GetFollowers safe {
     }
   }
 }
+
+model ErrorCode enum {
```

Comment:
BTW if we expand this enum, will it require a minor version bump? Technically I think it does.

Reply:
We return `ErrorCode` enum as result of perform call. Expanding the enum could break client code see:

```
switch (error.code) {
    case 'Unauthenticated':
        // handle unauthenticated error
        break;
    case 'Unauthorized':
        // handle unauthorized error
        break;
    default: // unknown error code values
        throw new Error('Not implemented');
}
```

I would bump major version of ErrorCode enum.

---

If the enum is used only in inputs minor version bump is sufficient.

Reply:
I added relevant Twitter error codes to the profile.